modules/rosa-hcp-sgs-and-vpce.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sgs-and-vpce_{context}"]
+= Configuring AWS security groups and privatelink connections
+
+After creating your VPC, create your AWS security groups and VPC endpoints.
+
+.Procedure
+
+. Create the AWS security group by running the following command:
++
+[source,terminal]
+----
+$ aws ec2 create-security-group \
+        --group-name allow-inbound-traffic 
+        --description "allow inbound traffic" 
+        --vpc-id $VPC_ID
```

Comment:
```suggestion
        --vpc-id <vpc_id>
```
`$VPC_ID` looks like a variable name. If this is a user replaced variable, it should be `<vpc_id>`. If this is part of the script mentioned earlier, I think you should use a callout to explain that this value is related to the script run earlier and that it must be done this way.

Does that make sense?